ONSdigital/census-rm-docker-dev
# RMC-128 Set IAC Profile to Dev

## Motivation and Context
The IAC service should now only have defaults in its dev profile.

## What has Changed
- Added flag to set active profile to dev

## How to test
Build the IAC service branch and `make up`, the IAC service should start with the new dev config.

## Links
https://trello.com/c/b5Wr4lEK/639-rmc-128-generate-and-assign-census-style-uac-5
https://github.com/ONSdigital/iac-service/pull/26
https://github.com/ONSdigital/census-rm-case-service/pull/9
https://github.com/ONSdigital/census-rm-kubernetes/pull/33
https://github.com/ONSdigital/census-rm-terraform/pull/47
